### DISCUSSION TITLE
ECS Agent Dockerfile template

### DISCUSSION THREAD

**Discussion by skeller88** [dagster-io/dagster#19246] [Q&A]
I want to deploy Dagster on ECS, and need to set up the agent. I'm not finding an obvious Dockerfile to use as a template for how to set up our Dockerfile. [Would it be something like this without the CMD step?](https://github.com/AntonFriberg/dagster-project-example/blob/main/docker/Dockerfile)

[I'm seeing a similar question for Helm](https://github.com/dagster-io/dagster/discussions/18520).

**Comment by jmsanders** [comment]
Most users deploy the agent directly from [an image we maintain](https://hub.docker.com/r/dagster/dagster-cloud-agent/tags) but if you want to customize it, the[ image layers](https://hub.docker.com/layers/dagster/dagster-cloud-agent/latest/images/sha256-fc17d77df762affbd76ed95a96e657a0f7e3750d5b34e036f70a4bc13dc5ab81?context=explore) that were used to build that image are a good reference.
